src/main/java/com/java_essential/varycheva/essential/homework/hw1/task1/Main.java

```diff
@@ -0,0 +1,19 @@
+package com.java_essential.varycheva.essential.homework.hw1.task1;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        double side1 = scan.nextDouble();
+        double side2 = scan.nextDouble();
+
+        if (side1 <= 0 || side2 <= 0) {
```

Comment:
```suggestion
       if ((side1 <= 0) || (side2 <= 0)) {
```
NOTE: to avoid unexpected mistakes more better to change this expression according to the variant bellow
 ` if ( (side1 <= 0) || (side2 <= 0) ) {`

Reply:
Fixed